TileStache/Goodies/VecTiles/transform.py

```diff
@@ -943,6 +945,10 @@ def exterior_boundaries(feature_layers, base_layer,
     """
     layer = None
 
+    # don't start processing until the start zoom
+    if zoom < start_zoom:
+        return layer
+
```

Comment:
Ah, perhaps I'm trying to be too clever there...

`None` appears to behave like "the lowest number", so `zoom < None` will always be false. But that's not clear from the code, so perhaps I'll just make the default `start_zoom` zero.